hypothesis-python/src/hypothesis/core.py

```diff
@@ -292,18 +310,22 @@ def accept(test):
 
 
 def reproduce_failure(version: str, blob: bytes) -> Callable[[TestFunc], TestFunc]:
-    """Run the example that corresponds to this data blob in order to reproduce
-    a failure.
-
-    A test with this decorator *always* runs only one example and always fails.
-    If the provided example does not cause a failure, or is in some way invalid
-    for this test, then this will fail with a DidNotReproduce error.
-
-    This decorator is not intended to be a permanent addition to your test
-    suite. It's simply some code you can add to ease reproduction of a problem
-    in the event that you don't have access to the test database. Because of
-    this, *no* compatibility guarantees are made between different versions of
-    Hypothesis - its API may change arbitrarily from version to version.
+    """
+    Run the example corresponding to the binary ``blob`` in order to reproduce a
+    failure.
+
+    A test decorated with |@reproduce_failure| always runs exactly one example,
+    which is expected to cause a failure. If the provided ``blob`` does not
+    cause a failure, Hypothesis will raise |DidNotReproduce|.
```

Comment:
I think somewhere here we could say that the blob is a serialized version of the choice sequence, just like we'd store in the database?  I don't recall whether we have a standard link target in the docs for that yet; it'd be a good candidate for an explanation page.